docs/bundles/SyliusPromotionBundle/action_applicator.rst

```diff
@@ -10,19 +10,17 @@ Actions can be created by implementing ``PromotionActionCommandInterface``. This
 
 Actions have to be defined as services and have to use the tag named ``sylius.promotion_action`` with the attributes ``type`` and ``label``.
 
-As ``SyliusPromotionBundle`` is totally independent, it does not provide some actions out of the box. Great examples of actions are provided by ``Sylius/Standard-Edition``.
+As ``SyliusPromotionBundle`` is totally independent, it does not provide some actions out of the box. Great examples of actions are provided by ``Sylius/Sylius-Standard``.
```

Comment:
`Great examples of actions are provided by ``Sylius/Sylius-Standard``.` Is it really true?